editor/src/messages/portfolio/portfolio_message_handler.rs

```diff
@@ -91,6 +93,7 @@ impl MessageHandler<PortfolioMessage, (&InputPreprocessorMessageHandler, &Prefer
 					responses.add(DocumentMessage::ClearLayerTree);
 					let hint_data = HintData(vec![HintGroup(vec![])]);
 					responses.add(FrontendMessage::UpdateInputHints { hint_data });
+					self.menu_bar_message_handler.process_message(MenuBarMessage::SendLayout, responses, false);
```

Comment:
Note that, to avoid unnecessary UI updates, this only really needs to be called:
- PortfolioMessage::DeleteDocument after setting self.active_document_id = None;
- PortfolioMessage::DestroyAllDocuments after setting self.active_document_id = None;
- PortfolioMessage::SetActiveDocument after setting self.active_document_id = Some(...)
These are the only times that the active document changes.

You can also use `responses.add(MenuBarMessage::SendLayout);` which is more concise and less error prone.